src/Parser/Warrior/Arms/Modules/Features/SpellUsable.js

```diff
@@ -0,0 +1,40 @@
+import SPELLS from 'common/SPELLS';
+import CoreSpellUsable from 'Parser/Core/Modules/SpellUsable';
+
+const TACTICIAN_ID = SPELLS.TACTICIAN.id;
+const MORTAL_STRIKE_ID = SPELLS.MORTAL_STRIKE.id;
+const COLOSSUS_SMASH_ID = SPELLS.COLOSSUS_SMASH.id;
+
+class SpellUsable extends CoreSpellUsable {
```

Comment:
You don't need to change this now but in the future you might want a tactician module that tracks wasted procs. When doing that it would be appropriate to also move handling of the resetting to said module.